addon/components/bs-collapse.js

```diff
@@ -92,6 +93,29 @@ export default class Collapse extends Component {
    */
   expandedSize = null;
 
+  /**
+   * Calculates a hash for style attribute.
+   */
+  @computed('collapseDimension', 'collapseSize')
+  get cssStyle() {
+    if (isNone(this.collapseSize)) {
+      return {};
+    }
+
+    if (this.transitionDuration) {
+      const transitionDurationS = this.transitionDuration / 1000;
+
+      return {
+        [this.collapseDimension]: `${this.collapseSize}px`,
+        transition: `${this.collapseDimension} ${transitionDurationS}s ease`,
```

Comment:
I did this because the css-duration must/should be kept in sync with components transition-duration and figured this would be an attempt to avoid that. But yes not really optimal because of ease, but also css-property prefixing might be an issue...

Reply:
Yeah, you definitely have a valid concern here! @jelhan brought this up a while ago: https://github.com/kaliber5/ember-bootstrap/issues/839

But we should tackle this in a separate PR at least, and for all components where this applies!